src/workspace/WorkspaceContent.jsx

```diff
@@ -17,6 +17,10 @@ const propTypes = {
    * Child node content to be displayed within the content region.
    */
   children: PropTypes.node,
+  /**
+   * Text to be displayed as the title of the workspace content.
```

Comment:
```suggestion
   * Text to be displayed as the title of the workspace content.
   * Special Note: this prop is optional and should be used with caution. If this prop is not provided, the workspace content title is injected by default using the label specified in `WorkspaceItem` _(recommended without additional guidance)_. Providing this prop will override the default text and will need to follow proper accessibility guidelines.
```

Reply:
Thinking that this special note can be moved to a standout/notice on a future PR when we provide full documentation. We then can replace this with something like _"See [xyz...] for usage instructions."_ at that time.